webui/react/src/pages/F_ExpList/glide-table/contextMenu.tsx

```diff
@@ -66,8 +68,10 @@ export const TableContextMenu: React.FC<TableContextMenuProps> = ({
       }}>
       <ExperimentActionDropdown
         experiment={experiment}
+        link={link}
         makeOpen={open}
         onComplete={onComplete}
+        onLink={handleClose}
```

Comment:
I think this is the only place where we assign a value for `onLink`; shouldn't it be possible to call `onComplete` or `onVisibleChange` (or neither since after clicking the link we just nav to another page)?

Reply:
Update: I forgot the new tab links - so we should call something but I don't know that it needs to be a special `onLink`

Reply:
Yeah I don't have the best naming capability lol. Do you have a suggestion on a name? `onClick`?

To describe it, it's basically a call to make upon completing a click on the context menu but waiting on the callback of the action...
- `onComplete` => `onAsyncComplete` and `onLink` => `onComplete`?
- `onLink` => `onClick`?

🤷

Reply:
In this case I might have reused onComplete, but I think I'm OK with onLink

Reply:
the `onComplete` for `ExperimentActionDropdown` makes a fetch, not the worse but this way we can avoid making an unnecessary API call upon just clicking on a link...